### DISCUSSION TITLE
Daily production log

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#9727] [Feature Requests]
Hello guys, 

Here I walk around with some tests until I came up with an idea, is it possible to do something like that in directus, something like a daily production record per operation.. :) 

I don't know at all if something like that fits in the project (Directus) but... 

Using the buttons for external links, open the side window (which is brutal) and there you have a daily production log.

Here's the idea :)

![Production_Register](https://user-images.githubusercontent.com/81326296/141328845-4c2fa47a-a840-41b0-95fd-e7788f75fdb6.jpg)

**Comment by rijkvanzanten** [comment]
Interesting! To me, it almost sounds like your "production_cut" would be a one-to-many field, but instead of having the inline rows, it's just a button to open the drawer (which would effectively be the same as the current "select existing". Does that sound about right? 🤔

**Reply by hugopvl** [reply]
That's right, but instead of selecting, the possibility of inserting records, with automatic dates if inserted… and that's a list of daily records…

**Reply by rijkvanzanten** [reply]
Ha that sounds like a hyper-specific use case, but a cool one indeed! 

I'm trying to find a more wider definition of exactly how it could work, without having it tied to the exact use case. Would it be fair to say it could be treated as an inline "row-style" insert interface for one-to-many records, but it opens in a drawer?

**Reply by hugopvl** [reply]
that's right, all the information in the records is kept in the drawer for consultation by "order" the total is shown in front of each "operation" but in these types of situations I can only fit them into production systems... that fits loosely...

**Reply by hugopvl** [reply]
@rijkvanzanten but I also made another sketch, which I manage to fit in various situations… so give me your opinion!?  🤔

![image](https://user-images.githubusercontent.com/81326296/141362873-e6638d24-c119-4e0c-97d1-e740560d355f.png)

**Reply by rijkvanzanten** [reply]
Ahh nice. What would be the technical back-end of this? I'd imagine it's not "just" a checkbox right? How does this tie in with the example in the OP?

**Reply by hugopvl** [reply]
well this is a button, with a menu and with the possibility of inserting options, such as "confirmed", "accepted", "rejected", "cancelled" each of these options with a specific color... and even with the possibility of inserting a date (thinking about planning systems, or approval systems or document management) but the issue of the date is that I don't know how to do it 😁🤔

**Reply by rijkvanzanten** [reply]
Ahh I see, so it's almost like a combined date input + dropdown, where you select one of the "states" (like _approved_, or _rejected_) and a date for when that state should be in effect?

**Reply by hugopvl** [reply]
a practical case, I submit a budget, for a project, which has to be approved by day X and the finance department has to approve or reject it, but it is registered if it was before or after the scheduled date, in a planning system, I do planning with several operations, each one has a scheduled date after or is approved or rejected, on or off the due date, so in any case it is possible to know if we are going to have delays, for example... this, together with the notifications, I think it was fine 🙂

**Reply by benhaynes** [reply]
A perfect extension... and will be easier to manage/install once the Marketplace is live. But this seems too specific to be in Core.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
